### ISSUE TITLE
Borders and backgrounds have precision issues.

### ISSUE THREAD

**Issue by DEF7** [vurtun/nuklear#479]
Playing with the SDL/GL3 demo I noticed a gap in the top row of pixels on the popup examples. It looks as though the pixels are there but only partial alpha. Is this the same half-pixel issue other people have mentioned? I changed the background to 0,255,0 just to make it more visible, but I first noticed it against the red button/rect. You can see that underlying text shows through too.

![nuklearpopupbug](https://user-images.githubusercontent.com/11559157/28045847-895d0748-6594-11e7-800f-806cf2153659.png)

**Comment by DEF7** [comment]
Ok here's an update. I tried disabling anti-aliasing, which solves the issue but now borders are erratic - specifically on windows/panels which are supposed to have a thickness of 2.0 pixels, and they fluctuate between 1-2 pixel thickness when resizing the window.

I did some more digging and found 'nk_draw_list_stroke_line()' which is responsible for rendering the borders. The culprit with the panel borders was the nk_vec2_add() with the 0.5f offsets. Removing this offset solves the window border thickness behaving erratically. With antialiasing on the result is still the same with the top of the tooltips having a gap, so that's coming from elsewhere.

However, when the offsets are removed the menus/popups don't draw their borders properly anymore, they are either missing horizontal borders or vertical, sometimes both. I tried disabling clipping via the argument to nk_command_buffer_init() inside the nk_nonblock_begin() function, but the borders are still missing. I guess this is why the 0.5 offsets were there to begin with?

So now I'm trying to track down wth is going on with popups that is causing their borders to be problematic and require the 0.5 offset to begin with.
